lib/ModelMixins/GeojsonMixin.ts

```diff
@@ -794,6 +813,9 @@ function GeoJsonMixin<T extends Constructor<Model<GeoJsonTraits>>>(Base: T) {
         }
       ];
 
+      // _catalogItem property is needed for some feature picking functions (eg FeatureInfoMixin)
+      (feature as any)._catalogItem = this;
+
```

Comment:
This is overwriting the `feature` import from `"@turf/helpers"`

Can this be done generically at the bottom of `forceLoadMapItems`? - for example

```ts
        this._dataSource?.entities.values.map(
          entity => ((entity as any)._catalogItem = this)
        );
```